app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -31,4 +43,62 @@ class StateFragmentPresenter @Inject constructor(
   fun setAudioFragmentVisible(isVisible: Boolean) {
     getStateViewModel().setAudioFragmentVisible(isVisible)
   }
+
+  private fun showInputInteractions() {
```

Comment:
Change this function to directly accept data, rather than creating data in this function.
This will help us in final implementation and we can directly pass data to the function and then it will manage it on its own.

Reply:
created dummy method to get data and separated this function for implementation